src/utils/user_profiles.py

```diff
@@ -20,7 +18,9 @@ def get_user_profiles(usernames: list, auth_token):
         'version': '1.1',
         'params': [usernames]
     }
-    headers = {'Authorization': auth_token}
+    headers = {}
+    if auth_token is not None:
```

Comment:
I take it that the profile service supports anonymous queries?

Reply:
yes.